CHANGELOG.next.asciidoc

```diff
@@ -42,6 +42,8 @@ https://github.com/elastic/beats/compare/v7.0.0-rc1...master[Check the HEAD diff
 *Filebeat*
 
 - Don't apply multiline rules in Logstash json logs. {pull}11346[11346]
+- Fix coredns image in docs.asciidoc for docs build. {pull}11460[11460] {pull}11461[11461]
```

Comment:
Minor issue with the cherry-pick. It gave you more than you wanted.

Reply:
Interesting. How can the script pickup another PR? I will remove it

Reply:
I think it's caused by git and not the script.